zyte_common_items/components.py

```diff
@@ -459,6 +464,11 @@ def to_scrapy(self, callback, **kwargs):
         )
 
 
+@attrs.define(kw_only=True)
+class Url(Item):
+    url: str = attrs.field(converter=url_to_str)
```

Comment:
Since it is rendered in the docs, it would be nice to provide some text, even if this case is extremely self-explaining.

```suggestion
class Url(Item):
    """A URL."""
    
    # : URL.
    url: str = attrs.field(converter=url_to_str)
```